src/epics/data.ts

```diff
@@ -518,8 +518,10 @@ const saveAssociationsActive: Epic = (action$, store) => action$.ofType(types.sa
     const pendingChanges = (state.view.pendingDataChanges[
         action.payload.bcNames[0]
     ] || {})
+    let params = {}
+    action.payload.bcNames.forEach(bcName => params = {...params, _bcName: bcName})
```

Comment:
1) Only the last bcName will be applied to params
```tsx
let params = {}
['bc1', 'bc2', 'bc3'].forEach(item => params = { ...params, _bcName: item })
console.log(params) // {_bcName: "bc3"}
```
2) You generally do not want to recreate an object inside `forEach` call when you can simply assign the field:
```tsx
action.payload.bcNames.forEach(bcName => {
    params._bcName = bcName
})
```
3) Curly bracers are mandatory for any assignments

Reply:
fixed

Reply:
It's still strangely iterates over an array to set the value once.
```tsx
const params: GetParamsMap = {}
action.payload.bcNames.forEach(bcName => {params._bcName = bcName})
```
What is happening here? Why you always use only the last bcName if there are multiple of them and if for some reason you do need the last one, why are you not accessing it directly?

Reply:
Fixed. Take to param only last bcName from array.
```
const bcNames = ['a','b','c']
const params = bcNames.length ? {_bcName: bcNames.pop()} : {}
console.log(params) // {_bcName: "c"}
```